README.md

```diff
@@ -39,31 +39,25 @@ EXAMPLES
 
 INSTALLATION AND CONFIGURATION
 ------------------------------
-
-
-1. Unzip the tarball to a directory of your choice, typically `/usr/local/`
-2. A new directory called easy-transform-metadata-<version> will be created
-3. Add the command script to your `PATH` environment variable by creating a symbolic link to it from a directory that is
-   on the path, e.g. 
-   
-        ln -s /usr/local/easy-transform-metadata-<version>/bin/easy-transform-metadata /usr/bin
-
-
-
-General configuration settings can be set in `cfg/application.properties` and logging can be configured
-in `cfg/logback.xml`. The available settings are explained in comments in aforementioned files.
-
+Currently this project is build only as an RPM package for RHEL7/CentOS7 and later. The RPM will install the binaries to
```

Comment:
@janvanmansum is this also applied in all the other projects? At least it isn't in `easy-module-archetype`, as I just created a new project together with @PaulBoon.

Besides, does this mean that the `bin.xml` can be removed in all projects?

For both points, if they're not yet done or cannot be done immediately, please create one or more issues for them. Thanks!

Reply:
I will update the template README in the archetype.

As for `bin.xml`, it depends on whether we still want to be able to build a `tar.gz` file. Currently, I think building the `tar.gz` is turned off in the build, as we only use the `rpm` anyway. A reason to the `bin.xml` could be to have some minimal support for non-RPM-based systems, in case anybody out there wants to install our software.

Reply:
We will make a decision about this later. This PR doesn't have to wait for that.